README.md

```diff
@@ -39,7 +39,7 @@ git clone https://github.com/Dmambo/Portfolio-setup-and-mobile-firs.git**
 -![alt text](screenshot.JPG "here is the screenshot)
 
 #Live version
--live website link (https://github.com/Dmambo/Dmambo.github.io)
+-live website link [click here for site](https://github.com/Dmambo/Dmambo.github.io)

```

Comment:
- Good job updating your readme, But the link to the live app redirects me to your repo and not the live link as it should. Please check it and update it with the correct link.